### ISSUE TITLE
Resetting scrap settings in LethalQuantities mod

### ISSUE THREAD

**Issue by MorsusP** [Major-Scott/TerminalPlus#4]
Funnily enough, I found another problem

This mod somehow resets the scrap settings in the **LethalQuantities** mod, meaning that whatever you adjust, everything goes back to vanilla values. Maybe it's a **LethalQuantities** mod problem I don't know exactly, but it's probably not likely^_^

**Comment by Major-Scott** [comment]
Yeah sadly that must be something on LQ's side; for the moons I'm doing my own custom information and settings but all the other tools/scrap settings I'm grabbing and printing directly from the main game :/
